<?xml version="1.0" encoding="utf-8"?>
<!-- This URDF was automatically created by SolidWorks to URDF Exporter! Originally created by Stephen Brawner (brawner@gmail.com) 
     Commit Version: 1.6.0-1-g15f4949  Build Version: 1.6.7594.29634
     For more information, please see http://wiki.ros.org/sw_urdf_exporter -->
<robot
  name="mim_diffdrive">
  <link
    name="base_link">
    <inertial>
      <origin
        xyz="-2.20799811772935E-15 -0.00101374975809105 0.212277859061032"
        rpy="0 0 0" />
      <mass
        value="33.5973988123837" />
      <inertia
        ixx="1.01936792689011"
        ixy="-2.87646051602348E-07"
        ixz="-3.0761567096847E-16"
        iyy="2.14070538797161"
        iyz="4.23705224594669E-16"
        izz="3.05624852691436" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mim_robots/meshes/diffdrive/base_link.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.529411764705882 0.549019607843137 0.549019607843137 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mim_robots/meshes/diffdrive/base_link.STL" />
      </geometry>
    </collision>
  </link>
  <link
    name="PW1">
    <inertial>
      <origin
        xyz="6.93889390390723E-16 2.019218126037E-15 -0.00999999999999998"
        rpy="0 0 0" />
      <mass
        value="0.0290199529351017" />
      <inertia
        ixx="1.74803544962354E-05"
        ixy="-1.39270131587091E-52"
        ixz="-2.44290946982965E-34"
        iyy="1.74803544962354E-05"
        iyz="8.83998436242655E-37"
        izz="3.31776708089229E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mim_robots/meshes/diffdrive/PW1.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.109803921568627 0.109803921568627 0.109803921568627 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mim_robots/meshes/diffdrive/PW1.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="PW1_J"
    type="continuous">
    <origin
      xyz="0 -0.200513749758132 0.0375000000000016"
      rpy="1.57079632679489 1.5707963267949 0" />
    <parent
      link="base_link" />
    <child
      link="PW1" />
    <axis
      xyz="0 0 1" />
  </joint>
  <link
    name="PW2">
    <inertial>
      <origin
        xyz="7.49400541621981E-16 -4.37497260641351E-15 0.00999999999999998"
        rpy="0 0 0" />
      <mass
        value="0.0290199529351017" />
      <inertia
        ixx="1.74803544962354E-05"
        ixy="-7.58567881730871E-36"
        ixz="-2.44357068540199E-34"
        iyy="1.74803544962354E-05"
        iyz="4.84891419714193E-37"
        izz="3.31776708089229E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mim_robots/meshes/diffdrive/PW2.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.109803921568627 0.109803921568627 0.109803921568627 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mim_robots/meshes/diffdrive/PW2.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="PW2_J"
    type="continuous">
    <origin
      xyz="0 0.198486250241876 0.0374999999999999"
      rpy="1.57079632679489 1.5707963267949 0" />
    <parent
      link="base_link" />
    <child
      link="PW2" />
    <axis
      xyz="0 0 1" />
  </joint>
  <link
    name="C1">
    <inertial>
      <origin
        xyz="-0.00657113223306066 -0.0285143133296709 -8.33960520085775E-09"
        rpy="0 0 0" />
      <mass
        value="0.133845469700625" />
      <inertia
        ixx="9.89825054141292E-05"
        ixy="-1.89872436567926E-05"
        ixz="7.537355636018E-12"
        iyy="4.2469650939329E-05"
        iyz="2.22958763736225E-11"
        izz="9.72073954309255E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mim_robots/meshes/diffdrive/C1.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.529411764705882 0.549019607843137 0.549019607843137 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mim_robots/meshes/diffdrive/C1.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="C1_J"
    type="continuous">
    <origin
      xyz="0.337499999999999 -0.178513749758133 0.124000000000001"
      rpy="1.57079632679489 0 0.0264866109494407" />
    <parent
      link="base_link" />
    <child
      link="C1" />
    <axis
      xyz="0 1 0" />
    <dynamics friction="0.7" />
  </joint>
  <link
    name="C1W">
    <inertial>
      <origin
        xyz="0 0 -5.55111512312578E-17"
        rpy="0 0 0" />
      <mass
        value="0.0290199529351017" />
      <inertia
        ixx="1.74803544962354E-05"
        ixy="5.14525307313489E-23"
        ixz="1.23325569560236E-21"
        iyy="1.75243625248463E-05"
        iyz="8.29982673807421E-07"
        izz="3.3133662780312E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mim_robots/meshes/diffdrive/C1W.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.109803921568627 0.109803921568627 0.109803921568627 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mim_robots/meshes/diffdrive/C1W.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="C1W_J"
    type="continuous">
    <origin
      xyz="-0.0210069660112502 -0.0865 0"
      rpy="-3.0886194316909 0 -1.5707963267949" />
    <parent
      link="C1" />
    <child
      link="C1W" />
    <axis
      xyz="0 0.0529484501323336 0.998597246956241" />
  </joint>
  <link
    name="C2">
    <inertial>
      <origin
        xyz="-0.00657113223306055 -0.0285143133296707 -8.3396052563689E-09"
        rpy="0 0 0" />
      <mass
        value="0.133845469700627" />
      <inertia
        ixx="9.89825054141297E-05"
        ixy="-1.89872436567927E-05"
        ixz="7.53735573194448E-12"
        iyy="4.24696509393292E-05"
        iyz="2.22958766818366E-11"
        izz="9.7207395430926E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mim_robots/meshes/diffdrive/C2.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.529411764705882 0.549019607843137 0.549019607843137 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mim_robots/meshes/diffdrive/C2.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="C2_J"
    type="continuous">
    <origin
      xyz="0.337500000000001 0.176486250241866 0.123999999999999"
      rpy="1.57079632679489 0 0.0264866109494407" />
    <parent
      link="base_link" />
    <child
      link="C2" />
    <axis
      xyz="0 1 0" />
    <dynamics friction="0.7" />
  </joint>
  <link
    name="C2W">
    <inertial>
      <origin
        xyz="0 -5.55111512312578E-17 0"
        rpy="0 0 0" />
      <mass
        value="0.0290199529351017" />
      <inertia
        ixx="1.74803544962354E-05"
        ixy="6.70982601326168E-23"
        ixz="1.50398684371662E-21"
        iyy="1.75243625248463E-05"
        iyz="8.29982673807421E-07"
        izz="3.3133662780312E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mim_robots/meshes/diffdrive/C2W.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.109803921568627 0.109803921568627 0.109803921568627 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mim_robots/meshes/diffdrive/C2W.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="C2W_J"
    type="continuous">
    <origin
      xyz="-0.0210069660112502 -0.0865 0"
      rpy="-3.0886194316909 0 -1.5707963267949" />
    <parent
      link="C2" />
    <child
      link="C2W" />
    <axis
      xyz="0 0.0529484501323336 0.998597246956241" />
  </joint>
  <link
    name="C3">
    <inertial>
      <origin
        xyz="-0.00657113223306055 -0.0285143133296708 -8.33960522861332E-09"
        rpy="0 0 0" />
      <mass
        value="0.133845469700626" />
      <inertia
        ixx="9.89825054141295E-05"
        ixy="-1.89872436567928E-05"
        ixz="7.53735573914426E-12"
        iyy="4.24696509393292E-05"
        iyz="2.22958765688636E-11"
        izz="9.7207395430926E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mim_robots/meshes/diffdrive/C3.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.529411764705882 0.549019607843137 0.549019607843137 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mim_robots/meshes/diffdrive/C3.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="C3_J"
    type="continuous">
    <origin
      xyz="-0.337499999999999 0.176486250241869 0.123999999999999"
      rpy="1.57079632679489 0 0.0264866109494407" />
    <parent
      link="base_link" />
    <child
      link="C3" />
    <axis
      xyz="0 1 0" />
    <dynamics friction="0.7" />
  </joint>
  <link
    name="C3W">
    <inertial>
      <origin
        xyz="0 0 -5.55111512312578E-17"
        rpy="0 0 0" />
      <mass
        value="0.0290199529351017" />
      <inertia
        ixx="1.74803544962354E-05"
        ixy="6.70982601325397E-23"
        ixz="1.50398684371664E-21"
        iyy="1.75243625248463E-05"
        iyz="8.29982673807421E-07"
        izz="3.3133662780312E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mim_robots/meshes/diffdrive/C3W.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.109803921568627 0.109803921568627 0.109803921568627 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mim_robots/meshes/diffdrive/C3W.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="C3W_J"
    type="continuous">
    <origin
      xyz="-0.02100696601125 -0.0865 0"
      rpy="-3.0886194316909 0 -1.5707963267949" />
    <parent
      link="C3" />
    <child
      link="C3W" />
    <axis
      xyz="0 0.0529484501323336 0.998597246956241" />
  </joint>
  <link
    name="C4">
    <inertial>
      <origin
        xyz="-0.00657113223306061 -0.0285143133296709 -8.33960522861332E-09"
        rpy="0 0 0" />
      <mass
        value="0.133845469700625" />
      <inertia
        ixx="9.89825054141289E-05"
        ixy="-1.89872436567925E-05"
        ixz="7.53735568789876E-12"
        iyy="4.24696509393288E-05"
        iyz="2.22958765092536E-11"
        izz="9.72073954309252E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mim_robots/meshes/diffdrive/C4.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.529411764705882 0.549019607843137 0.549019607843137 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mim_robots/meshes/diffdrive/C4.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="C4_J"
    type="continuous">
    <origin
      xyz="-0.337500000000001 -0.178513749758131 0.123997864045001"
      rpy="1.57079632679489 0 0.0264866109494407" />
    <parent
      link="base_link" />
    <child
      link="C4" />
    <axis
      xyz="0 1 0" />
    <dynamics friction="0.7" />
  </joint>
  <link
    name="C4W">
    <inertial>
      <origin
        xyz="0 -5.55111512312578E-17 -2.22044604925031E-16"
        rpy="0 0 0" />
      <mass
        value="0.0290199529351017" />
      <inertia
        ixx="1.74803544962354E-05"
        ixy="6.70982601325397E-23"
        ixz="1.50398684371664E-21"
        iyy="1.75243625248463E-05"
        iyz="8.29982673807422E-07"
        izz="3.3133662780312E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mim_robots/meshes/diffdrive/C4W.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.109803921568627 0.109803921568627 0.109803921568627 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mim_robots/meshes/diffdrive/C4W.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="C4W_J"
    type="continuous">
    <origin
      xyz="-0.02100696601125 -0.0864999999999998 0"
      rpy="-3.0886194316909 0 -1.5707963267949" />
    <parent
      link="C4" />
    <child
      link="C4W" />
    <axis
      xyz="0 0.0529484501323336 0.998597246956241" />
  </joint>
  <link
    name="LIDAR">
    <inertial>
      <origin
        xyz="0.0121635546480126 -0.0287648885617243 0.000482127716282299"
        rpy="0 0 0" />
      <mass
        value="0.0730843683225071" />
      <inertia
        ixx="1.9581193027534E-05"
        ixy="-4.28353442325277E-07"
        ixz="1.03773440001469E-07"
        iyy="4.14178206744278E-05"
        iyz="-9.42533290571976E-09"
        izz="2.41241313339889E-05" />
    </inertial>
    <visual>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mim_robots/meshes/diffdrive/LIDAR.STL" />
      </geometry>
      <material
        name="">
        <color
          rgba="0.752941176470588 0.752941176470588 0.752941176470588 1" />
      </material>
    </visual>
    <collision>
      <origin
        xyz="0 0 0"
        rpy="0 0 0" />
      <geometry>
        <mesh
          filename="package://mim_robots/meshes/diffdrive/LIDAR.STL" />
      </geometry>
    </collision>
  </link>
  <joint
    name="LIDAR_J"
    type="fixed">
    <origin
      xyz="0.321103295710635 -0.000963772085925339 0.384500000000008"
      rpy="1.57079632679489 0 -3.14159265358979" />
    <parent
      link="base_link" />
    <child
      link="LIDAR" />
    <axis
      xyz="0 0 0" />
  </joint>
</robot>
